src/frontend/data/sideEffects/updateLiveParticipants/index.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import fetchMock from 'fetch-mock';
```

Comment:
in the commit message
A new endpoint has been added to the video API which allows to update
the participants lists.
So, we need a frondent sideEffect matchimg it.
A convenient util wraps this sideEffect through functions that manages
both lists to keep them coherent.
=>
A new endpoint has been added to the video API which allows **updating**
the **participant** lists.
So, we need a frondent sideEffect **matching** it.
**Convenient** util wraps this sideEffect through functions that **manage**
both lists to keep them coherent.